assets/css/main.css

```
@@ -1116,3 +1116,65 @@ ul.post-archive{margin-left:0!important;}
 .error-wrap .message-manual {
     margin-bottom: 32px;
 }
+
+/* ====================== bookmark card ====================== 2019.09.07 */
+.kg-bookmark-container {
+	display: flex;
+	flex-wrap: wrap;
+	color: black;
+	text-decoration: none;
+	margin: 0;
+	border-radius: 5px;
+	overflow: hidden;
+	border: 1px solid #e5eff5 !important;
+	box-shadow: 0 0 1px rgba(0,0,0,.06),0 2px 6px rgba(0,0,0,.03);
+}
+
+.kg-bookmark-card a:hover {
+	color: black !important;
+}
+
+
+.kg-bookmark-content {    
+	flex-basis: 0;
+	flex-grow: 999;
+	min-width: 50%;
+	padding: 20px;
+	font-size: 1rem;
+	line-height: 1;
+	text-align: left;
+}
+
+.kg-bookmark-title {
+	font-weight: 600;
+	font-size: 1rem;
+}
+
+.kg-bookmark-description,
+.kg-bookmark-metadata {
+	margin-top: 12px;
```

```suggestion
	margin-top: .5em;
	line-height:1.5em;
	max-height: 3em;
	overflow-y: hidden;
```
I tend to use relative values for space such as the height to avoid responsive problems. 

The max-height and overflow-y will allow up to 2 lines of description which is used in the Ghost editor and I think it is better to cut everything more than to keep it a bit precise.

---

Agree, looks good. I did start to change from px to rem, but wasn't sure if it was required.